test/form/samples/pure-comments-multiple/main.js

```diff
@@ -0,0 +1,2 @@
+/*#__PURE__*//*#__PURE__*//*#__PURE__*//*#__PURE__*//*#__PURE__*/ removed();
```

Comment:
This tests for the multiple comment issue (I know, it is probably not very realistic, but still).